Purchases/Public/Purchases.swift

```diff
@@ -612,12 +615,13 @@ extension Purchases {
      * - Parameter data: Dictionary provided by the network. See https://docs.revenuecat.com/docs/attribution
      * - Parameter network: Enum for the network the data is coming from, see ``AttributionNetwork`` for supported
      * networks.
-     * - Parameter networkUserId: User Id that should be sent to the network. Default is the current App User Id.
+     * - Parameter maybeNetworkUserId: User Id that should be sent to the network. Default is the current App User Id.
      */
+    @available(*, deprecated, message: "Use the set<NetworkId> functions instead")
     public static func addAttributionData(_ data: [String: Any],
```

Comment:
is this 3rd version of the function still necessary? were we avoiding breaking changes with the argument labels?

Reply:
I was avoiding breaking changes, but really... maybe we should remove this? We are shipping a new major version.

Reply:
either that, or align the swift parameter labels with those from before and add an @objc version of the method name...kinda what we did with optional external parameter namings, or else one of the APIs had to break.

Reply:
I just took a look at this again, we can't get rid of it yet, we don't have a `setAppleSearchAdsData` method so we can't actually redirect folks to the `set<network>` methods